app/views/CreateDiscussionView/index.tsx

```diff
@@ -188,6 +181,14 @@ class CreateChannelView extends React.Component<ICreateChannelViewProps, ICreate
 								/>
 							</>
 						) : null}
+
+						<Button
+							testID='create-discussion-submit'
+							disabled={!this.valid()}
+							style={{ marginTop: 24 }}
```

Comment:
36 on Figma.
Is it inheriting 12 from another component?

Reply:
it was inheriting but not 12, my bad. Adjusted the styles to be clear.